azurerm/resource_arm_dns_zone.go

```diff
@@ -45,6 +45,29 @@ func resourceArmDnsZone() *schema.Resource {
 				Set:      schema.HashString,
 			},
 
+			"zone_type": {
```

Comment:
(as above) can we document these new fields? The documentation for this new resource can be found in `./website/docs/r/dns_zone.html.markdown`

Reply:
done.